documentation/access-request-management.md

```diff
@@ -0,0 +1,130 @@
+Bad requests, possibly due to improper access request definition, will respond with **status code 400** (to be implemented) <!-- TODO: implement -->
+When the access requested has been validated (to be implemented), but the storage fails, the response will have **status code 500**.
+
+### Example POST request
+
+This example creates an access request `ex:request` for the RP `https://example.pod.knows.idlab.ugent.be/profile/card#me`:
+
+```shell-session
+curl --location 'http://localhost:4000/uma/requests' \
+--header 'Authorization: https://example.pod.knows.idlab.ugent.be/profile/card#me' \
+--header 'Content-Type: text/turtle' \
+--data-raw '
+@prefix sotw: <https://w3id.org/force/sotw#> .
+@prefix odrl: <https://www.w3.org/ns/odrl/2/> .
+@prefix dcterms: <https://purl.org/dc/terms/> .
+@prefix dct: <https://purl.org/dc/terms/> .
+@prefix ex: <https://example.org/> .
+@prefix xsd: <https://www.w3.org/2001/XMLSchema#> .
+
+ex:request a sotw:EvaluationRequest ;
+      dcterms:issued "2025-08-21T11:24:34.999Z"^^xsd:datetime ;
+      sotw:requestedTarget <http://localhost:3000/resources/resource.txt> ;
+      sotw:requestedAction odrl:read ;
+      sotw:requestingParty <https://example.pod.knows.idlab.ugent.be/profile/card#me> ;
+      ex:requestStatus ex:requested .
+'
+```
+
+## Reading access requests
+
+To read policies, a single endpoint is currently implemented.
+This endpoint currently returns the list of access requests where the WebID provided in the `Authorization` header is marked as the requesting party.
+[See also](#coupling-of-target-to-ros).
+An example request to this endpoint is:
+
+```shell-session
+curl -X GET --location 'http://localhost:4000/uma/requests' \
+--header 'Authorization: https://example.pod.knows.idlab.ugent.be/profile/card#me'
+```
+
+## Updating access requests
+
+Updating policies can be done through a **PATC** request.
+The body must hold the content type `application/sparql-update`.
+The query can use the **DELETE/INSERT** statement to update properties of the access request.
+The example below shows how to update the access request's status from `requested` to `accepted`:
+
+```shell-session
+curl -X PATCH --location 'http://localhost:3000/' \
+--header 'Authorization: https://example.pod.knows.idlab.ugent.be/profile/card#me' \
+--header 'Content-Type: application/sparql-update' \
+--data-raw '
+PREFIX ex: <https://example.org/>
+PREFIX sotw: <https://w3id.org/force/sotw#>
+
+DELETE {
+    ?request ex:requestStatus ex:requested .
+} INSERT {
+    ?request ex:requestStatus ex:accepted . # change to `ex:denied` in order to deny
+} WHERE {
+    ?request sotw:requestedTarget <http://localhost:3000/resources/resource.txt> .
+}'
+```
+
+## Deleting access requests
+
+<!-- TODO: figure out -->
+
+## Future work
+
+### Discrepancies between [earlier descriptions](https://github.com/bramcomyn/loama/blob/feat/odrl/documentation/access_grants_vs_dsnp.md) and this implementation
```

Comment:
This will be the authorative source, so it is better that LOAMA will point to here instead of vice versa.